staff/alexgbor/count-words/scripts/countWords.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+function countWords(str) {
+    if (typeof str=="string" && str.length>0) {
+        var counter=1;
+        for (var i=0;i<str.length;i++) {
+            if (str[i]===" ") {
+                counter+=1;
+            }
+        }
+    return counter;
+    }
+    else {
+        throw Error("You must input a string.");
+    }
+}
```

Comment:
Tip: To count words yo could split the string into an array to have each word in an array position and then get the length of that array.

[split](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split)